assignment-section-05/generate_test_data.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+
+from classification_model.config.core import config, DATASET_DIR
+from classification_model.processing.data_manager import _load_raw_dataset
+
+
+def generate_data_data() -> None:
```

Comment:
@solegalli I downloaded the data as `raw.csv`, put it in the datasets directory, and then ran this script to generate a `test.csv` for use in the testing. My assumption is that we want to keep some part for testing (or can we assume train_test_split always gives the same rows for testing?)

Reply:
Students won't know where raw.csv comes from. And they would probably try to pick it up from the url directly when refactoring the code. So if you want to proceed with raw.csv, you will need to tell them to download it and save it somewhere early on in the assignment.

if we set the `random_state` in `train_test_split,` supposedly, it should always return the same train/test split.

Reply:
Agree this is too complex. I've updated the test code to just use `train_test_split` now. I'll update the lecture video to explain downloading the code and saving a single file: raw.csv

Reply:
I've removed the generate test data script